ui/src/file_upload.tsx

```diff
@@ -72,9 +74,17 @@ const
       minWidth: 80,
       boxSizing: 'border-box',
       height: 32,
+      display: 'inline-block'
+    },
+    asterisk: {
       $nest: {
-        '&:hover': {
-          cursor: 'pointer'
```

Comment:
This should stay in `uploadLabel` class to get the proper UX.

Reply:
Thanks for noticing!